### PR TITLE
fix: Update rendering of preamble and usage

### PR DESCRIPTION
**Pull request by tayloramurphy**
* Adds settings_preamble to the plugins page
* Adds it to render right after the settings header
* Moves the usage block _below_ the Settings which was it's original place.

### CODE REVIEW

**Comment by edgarrmondragon** [comment]
Duplicate of #911. Sorry about that @tayloramurphy 😅. Happy to review this one and close my own, though.

File: src/components/PluginSettingsSection.vue [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <template>
   <div>
     <p class="text-3xl py-4" id="settings">Settings</p>
+    <div
+      class="prose mt-3 p-2"
+      v-if="$page.plugins.settings_preamble"
+      v-html="$page.plugins.settings_preamble_rendered"
+    ></div>
```
Comment (edgarrmondragon) [inline src/components/PluginSettingsSection.vue:8]:
```suggestion
```

File: src/templates/Plugins.vue [hunk 2]
```diff
@@ -220,6 +215,11 @@
                 :plugin_type_plural="$page.plugins.pluginTypePlural"
                 :variant="$page.plugins.variant"
```
Comment (edgarrmondragon) [inline src/templates/Plugins.vue:216]:
```suggestion
                :variant="$page.plugins.variant"
                :preamble="$page.plugins.settings_preamble_rendered"
```

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]
The settings template already has a `preamble` prop that you can use

**Comment by tayloramurphy** [comment]
> Duplicate of #911. Sorry about that @tayloramurphy 😅. Happy to review this one and close my own, though.

I think I'm the one who should apologize, I think I made more work for you haha! Plus I moved some stuff around in this PR too 😅

**Review by edgarrmondragon — APPROVED** [review]
@tayloramurphy LGTM!

https://deploy-preview-914--meltano-hub.netlify.app/transformers/dbt-bigquery/#settings

**Pull request merged**